src/common.js

```diff
@@ -53,6 +53,20 @@ function ShellString(str) {
 }
 exports.ShellString = ShellString;
 
+// Return the home directory in a platform-agnostic way, with consideration for
+// older versions of node
+function getUserHome() {
```

Comment:
Why not just use [`os-homedir`](https://github.com/sindresorhus/os-homedir) instead? I don't really feel like we need to reinvent the wheel.